build.py

```diff
@@ -1,18 +1,71 @@
-import math
-import subprocess
 from pathlib import Path
 
+import argparse
+
 import fontmake.instantiator
 import fontTools.designspaceLib
 import ufo2ft
 import ufoLib2
 import vttLib
-import copy
 
 INPUT_DIR = Path("sources")
 OUTPUT_DIR = Path("build")
+VTT_DATA_FILE = (INPUT_DIR / "vtt_data" / "CascadiaCode.ttx")
+
+def step_set_font_name(n):
+    def _set(instance):
+        instance.info.familyName = n
+        # We have to change the style map family name because that's what
+        # Windows uses to map Bold/Regular/Medium/etc. fonts
+        instance.info.styleMapFamilyName = n
+    return _set
+
+def step_merge_glyphs_from_ufo(path):
+    def _merge(instance):
+        ufo = ufoLib2.Font.open(path)
```

Comment:
There's a possible optimization in here somewhere to only load extra UFOs once. Disk access is brutally slow especially on Windows.

Reply:
so: I explicitly did it that way first, but merging the glyphs in and compiling the resulting instance irreversibly changes the state in `ufo`

Reply:
(so it fails to compile when the decorated function happens again :smile:)

Reply:
Oops. One could do `inplace=False` or `instance.addGlyph(copy.deepcopy(ufo[glyph]))`.